sql/solutions/lesson1.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Introduction to SQL Solutions
+---
+# Lesson 1 solutions to exercises
+
+## L1.1 Show all the data available of all the people.
+
+```SQL
+SELECT *
+  FROM people;
+```
+
+##  L1.2 Show the name and Country of residence of all the people 23 years old or above.
+
+```SQL
+SELECT name, country_of_residence
+  FROM people
+ WHERE age >= 23;
+```
+
+##  L1.3 Show the name and age of all the people living in UK.
+
+```SQL
+SELECT name, age
+  FROM people
+ WHERE country_of_residence >= 'UK';
+```
+
+##  L1.4 Show the name and age of all the people not living in their birth place.
+
+```SQL
+SELECT name, age
+  FROM people
+ WHERE NOT country_of_birth = country_of_residence;
```

Comment:
This solution is correct, but there is an alternative which, at least to me, is easier to read. Can you add this one too?
Solution 2
SELECT name, age
  FROM people
 WHERE country_of_birth <> country_of_residence;